lambda-notes/src/Components/NoteView.js

```diff
@@ -0,0 +1,276 @@
+import React, { Component } from "react";
+import axios from "axios";
+import Modal from "react-responsive-modal";
+import PropTypes from "prop-types";
+import styled from 'styled-components';
+
+const Button = styled.button`
+border: none;
+background-color: #ccffff;
+font-size: 1em;
+margin: 1em;
+`
+
+const SubmitButton = styled(Button)`
+border-radius: 5px; //I don't like square buttons
+display: flex;
+background-color: #4dc3ff;
+color: white;
+font-size: 1em;
+margin: 1em;
+padding: 0.25em 1em;
+width: 30%;
+height: 40px;
+`
+
+const DeleteButton = styled(Button)`
+border-radius: 5px; //I don't like square buttons
+display: flex;
+background-color: #ff3300
+color: white;
+font-size: 1em;
+margin: 1em;
+padding: 0.25em 1em;
+width: 60%;
+height: 40px;
+`
+
+const NoButton = styled(Button)`
+border-radius: 5px; //I don't like square buttons
+display: flex;
+background-color: #4dc3ff;
+color: white;
+font-size: 1em;
+margin: 1em;
+padding: 0.25em 1em;
+width: 60%;
+height: 40px;
+`
+
+const NoteButtons = styled.div`
+display: flex;
+justify-content: flex-end;
+`
+
+const NoteHeader = styled.div`
+margin: 1em;
+`
+
+const NoteContainer = styled.div`
+background-color: #ccffff;
+width: 100%;
+`
+
+const NoteBody = styled.div`
+margin: 1em;
+font-size: 1.1rem;
+`
+
+const NoteTitle = styled.div`
+margin-top: 1em;
+font-weight: bold;
+font-size: 1.5rem
+`
+
+const EditForm = styled.div`
+width: 100%;
+background-color: #ccffff;
+`
+
+const Edithead = styled.div`
+margin: 1em;
+`
+
+const EditTitle = styled.div`
+margin: 1em;
+`
+
+const EditBody = styled.div`
+margin: 1em;
+`
+
+const ModalContainer = styled.div`
+display: flex;
+flex-direction: column;
+width: 100%;
+`
+
+const ButtonContainer = styled.div`
+display: flex;
+`
+
+//NoteView is the class that displays a single note. This class deploys the editNote() and deleteNote() functions, which live in the App.js file
+
+class NoteView extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      editing: false,
+      open: false
+    };
+  }
+
+  //Mounts fetched note from fetch() function to state
+  componentDidMount() {
+    window.scrollTo(0, 0);
+    const id = this.props.match.params.id;
+    console.log(id);
+    this.fetch(id);
+  }
+
+  //Fetches an individual note by id and sets its values to state of this component.
+  fetch = id => {
```

Comment:
method name itself should give future viewers a good idea of what it does, so maybe something like `fetchIndividualNote` might be better 🤔